tools/slice_image/slice_image.xml

```diff
@@ -0,0 +1,89 @@
+<tool id="slice_image" name="Slice Image" version="0.1"> 
+  <description>Slice image</description>
+  <requirements> 
+    <requirement type="package" version="0.14.0">scikit-image</requirement>
+    <requirement type="package" version="1.15.4">numpy</requirement>
+    <requirement type="package" version="1.1.0">scipy</requirement>
+  </requirements>
+  <command> 
+        <![CDATA[
+        mkdir ./out_dir &&
+
+        python "$__tool_directory__/slice_image.py" 
```

Comment:
please use single-quote ... `'`? Instead of double quotes. https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#command-formatting